.github/workflows/ci.yaml

```diff
@@ -0,0 +1,121 @@
+                env:
+                    PHPSTAN_ARGS: --error-format=checkstyle
+                run: make -is static-analysis | cs2pr
+
+    coding-standards:
+        name: Coding Standards
+        runs-on: ubuntu-20.04
+        strategy:
+            matrix:
+                php-version: [7.3]
+        steps:
+            -   uses: actions/checkout@v2
+            -   uses: shivammathur/setup-php@v2
+                with:
+                    coverage: none
+                    php-version: ${{ matrix.php-version }}
+                    tools: cs2pr
+
+            -   name: Cache dependencies installed with composer
+                uses: actions/cache@v1
+                with:
+                    path: ~/.composer/cache
+                    key: php-${{ matrix.php-version }}
+                    restore-keys: php-${{ matrix.php-version }}
+
+            -   name: Install dependencies with composer
+                run: COMPOSER_ARGS="--prefer-stable" make
+
+            -   name: Run squizlabs/php_codesniffer
+                env:
+                    PHPCS_ARGS: -q --no-colors --report=checkstyle
+                run: make -is cs | cs2pr
+
+    tests:
+        name: Tests
+        runs-on: ubuntu-20.04
+        strategy:
+            matrix:
+                php-version: [7.3, 7.4]
+                dependencies: ["", --prefer-lowest]
+
+        steps:
+            -   uses: actions/checkout@v2
+            -   uses: shivammathur/setup-php@v2
+                with:
+                    coverage: none
+                    php-version: ${{ matrix.php-version }}
+
+            -   name: Cache dependencies installed with composer
+                uses: actions/cache@v1
+                with:
+                    path: ~/.composer/cache
+                    key: php-${{ matrix.php-version }}-dependencies-${{ matrix.dependencies }}
+                    restore-keys: php-${{ matrix.php-version }}
+
+            -   name: Install dependencies with composer
+                run: COMPOSER_ARGS="--prefer-stable ${{ matrix.dependencies }}" make
+
+            -   name: Run tests
+                run: make test
+
+    coverage:
+        name: Tests Coverage
+        runs-on: ubuntu-20.04
+        strategy:
+            matrix:
+                php-version: [7.3]
+
+        steps:
+            -   uses: actions/checkout@v2
+            -   uses: shivammathur/setup-php@v2
+                with:
+                    coverage: pcov
+                    php-version: ${{ matrix.php-version }}
+
+            -   name: Cache dependencies installed with composer
+                uses: actions/cache@v1
+                with:
+                    path: ~/.composer/cache
+                    key: php-${{ matrix.php-version }}
+                    restore-keys: php-${{ matrix.php-version }}
+
+            -   name: Install dependencies with composer
+                run: COMPOSER_ARGS="--prefer-stable" make
```

Comment:
asi bych ty env vars nastavoval přes `env:` like this https://docs.github.com/en/free-pro-team@latest/actions/reference/environment-variables#about-environment-variables

je pak přehlednější co pouštíš ... ale je to jedno, rozhodně to není špatně :) jen nápad na improvement